dags/update_orphaning_dashboard_etl.py

```diff
@@ -100,3 +101,26 @@
         gcp_conn_id=gcp_conn_id,
     ),
 )
+
+# TODO remove this when the job writes to GCS directly
+gcs_sync = GKEPodOperator(
+    task_id="aws_s3_sync",
```

Comment:
nit: consider renaming the step so it's clear we are sync'ing between s3 and GCS (like the variable name)